bucket/brawl-crate.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "0.37",
+    "description": "BrawlBox/BrawlTools Fork, Wii File Editor.",
+    "homepage": "https://github.com/soopercool101/BrawlCrate/",
+    "url": "https://github.com/soopercool101/BrawlCrate/releases/download/v0.37/BrawlCrate.v0.37.x86.exe#/dl.7z",
+    "hash": "bfd95247075b468fc159f0b12b43dfcdc62ed38500394e7f0f7982d1229aeaa9",
+    "license": "Visable Source",
```

Seems to be MIT-licensed: https://github.com/soopercool101/BrawlCrate/blob/master/LICENSE

```suggestion
    "license": "MIT",
```